functests/utils/mcps/mcps.go

```diff
@@ -216,3 +218,19 @@ func WaitForProfilePickedUp(mcpName string, profile *performancev2.PerformancePr
 		return false
 	}, cluster.ComputeTestTimeout(10*time.Minute, runningOnSingleNode), 30*time.Second).Should(BeTrue(), "PerformanceProfile's %q MC was not picked up by MCP %q in time", profile.Name, mcpName)
 }
+
+func Delete(name string) error {
+	mcp := &machineconfigv1.MachineConfigPool{}
+	if err := testclient.Client.Get(context.TODO(), types.NamespacedName{Name: name}, mcp); err != nil {
```

Comment:
In general, I can do the empty object, but I think it is just the preferred way, to fetch an object before deletion, and it should not take any time, because we are fetching from the cache.

Reply:
works for me